Assets/Scripts/UI/Canvas/StateMachine/UIStateManager.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using Framework.PhoneCamera;
+using UnityEngine;
+
+using UI.Canvas.Introduction;
+using UI.Canvas.PhotoBookSystem;
+using UI.Canvas.PhotoTaking;
```

Comment:
```suggestion
using System;
using UnityEngine;

using Framework.PhoneCamera;
using UI.Canvas.Introduction;
using UI.Canvas.PhotoBookSystem;
using UI.Canvas.PhotoTaking;
```
Standaard namespaces en onze moeten gesplits zijn.